lib/govuk_app_config/configure.rb

```diff
@@ -8,15 +8,23 @@
 
   # We're misusing the `should_capture` block here to hook into raven until
   # there's a better way: https://github.com/getsentry/raven-ruby/pull/750
-  config.should_capture = Proc.new {
-    GovukStatsd.increment("errors_occurred")
+  GovukError.configure do |error_config|
```

Comment:
Does this need to be in the `Raven.configure` block?

Reply:
Nope. Good point!